<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunDialog</class>
 <widget class="QDialog" name="RunDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="ls_layout">
     <item>
      <widget class="QComboBox" name="ls_combobox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Blga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ClusteredClearing</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>338</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="popsize_layout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Population size</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="popsize_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>268</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="popsize_spinbox">
       <property name="statusTip">
        <string extracomment="Population size"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="pamnass_layout">
     <item>
      <widget class="QLabel" name="pamnass_label">
       <property name="text">
        <string>Chromosome number selected on PAM selection</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="pamnass_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>78</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="pamnass_spinbox">
       <property name="statusTip">
        <string extracomment="pamnass"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="alfar_layout">
     <item>
      <widget class="QComboBox" name="alfa_r_combo_box">
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>alfa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>r</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="alfa_r_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>295</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStackedWidget" name="alfa_r_stackw">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>23</height>
        </size>
       </property>
       <property name="statusTip">
        <string extracomment="alfa"/>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="alfa_r_page1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QSpinBox" name="alfa_spinbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>999</number>
           </property>
           <property name="value">
            <number>7</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="alfa_r_page2">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QDoubleSpinBox" name="r_dspinbox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="statusTip">
            <string extracomment="r"/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <double>1.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rtsnass_layout">
     <item>
      <widget class="QLabel" name="rtsnass_label">
       <property name="text">
        <string>Number of chromosome on RTS</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rtsnass_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>178</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="rtsnass_spinbox">
       <property name="statusTip">
        <string extracomment="rtsnass"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="evalnum_layout">
     <item>
      <widget class="QLabel" name="evalnum_label">
       <property name="text">
        <string>Number of fitness eval.</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="evalnum_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>228</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="evalnum_spinbox">
       <property name="statusTip">
        <string extracomment="numEvaluations"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="startrun_layout">
     <item>
      <widget class="QLabel" name="startrun_label">
       <property name="text">
        <string>Start run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="startrun_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>278</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="startrun_spinbox">
       <property name="statusTip">
        <string extracomment="startRun"/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>98</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="endrun_layout">
     <item>
      <widget class="QLabel" name="endrun_label">
       <property name="text">
        <string>End run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="endrun_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>308</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="endrun_spinbox">
       <property name="statusTip">
        <string extracomment="endRun"/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="fitness_groupbox">
     <property name="title">
      <string>Fitness function</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="fitness_layout">
        <item>
         <layout class="QHBoxLayout" name="fitnessclass_layout">
          <item>
           <widget class="QLabel" name="fitnessclass_label">
            <property name="text">
             <string>Fitness function class</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="fitnessclass_spacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>118</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="fitnessclass_combobox">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Periodic equal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Periodic unequal</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Hump function</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="path_layout">
          <item>
           <widget class="QLabel" name="path_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="path_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="statusTip">
             <string extracomment="Click to select path to hump function configuration file"/>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="popupMode">
             <enum>QToolButton::InstantPopup</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lsoptions_group">
     <property name="title">
      <string>Local searcher specific options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QStackedWidget" name="lsopt_stackw">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="lsopt_page1">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="lsopt_page1_layout">
            <item>
             <layout class="QHBoxLayout" name="itcalls_layout">
              <item>
               <widget class="QLabel" name="itcalls_spacer_2">
                <property name="text">
                 <string>Iterate calls</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="itcalls_spacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>188</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="itcalls_spinbox">
                <property name="statusTip">
                 <string extracomment="itCalls"/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="prefix">
                 <string/>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="lsopt_page1_vspacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="lsopt_page2">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="lsopt_page2_layout">
            <item>
             <layout class="QHBoxLayout" name="offsprings_layout">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Offspring number</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="offsprings_spacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>128</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="offsprings_spinbox">
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="maximum">
                 <number>9999</number>
                </property>
                <property name="singleStep">
                 <number>5</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="clearing_layout">
              <item>
               <widget class="QLabel" name="clearing_label">
                <property name="text">
                 <string>Clearing radius</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="clearing_spacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>118</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="clearing_spinbox">
                <property name="statusTip">
                 <string extracomment="clRadius"/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
                <property name="specialValueText">
                 <string/>
                </property>
                <property name="singleStep">
                 <double>0.050000000000000</double>
                </property>
                <property name="value">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="spacers_layout">
     <item>
      <spacer name="left_vspacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>88</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="right_vspacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>88</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <spacer name="buttons_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>206</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="button_box">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ls_combobox</tabstop>
  <tabstop>popsize_spinbox</tabstop>
  <tabstop>pamnass_spinbox</tabstop>
  <tabstop>alfa_r_combo_box</tabstop>
  <tabstop>r_dspinbox</tabstop>
  <tabstop>rtsnass_spinbox</tabstop>
  <tabstop>evalnum_spinbox</tabstop>
  <tabstop>startrun_spinbox</tabstop>
  <tabstop>endrun_spinbox</tabstop>
  <tabstop>fitnessclass_combobox</tabstop>
  <tabstop>path_button</tabstop>
  <tabstop>offsprings_spinbox</tabstop>
  <tabstop>clearing_spinbox</tabstop>
  <tabstop>itcalls_spinbox</tabstop>
  <tabstop>alfa_spinbox</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>RunDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>RunDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>584</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ls_combobox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>lsopt_stackw</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>411</x>
     <y>382</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>alfa_r_combo_box</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>alfa_r_stackw</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>34</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
